static/modes/python-mode.js

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2019, Sebastian Rath
```

Comment:
Monaco has support for python built-in: https://github.com/microsoft/vscode/tree/master/extensions/python

Is there a compelling reason to ship our own language definition or were you just not aware of this?

Locally I am able to entirely remove this file (and the reference in `static/panes/editor.js` and still get identical syntax highlighting in the browser.

Reply:
Thanks for the info! I wasn't aware of it

Reply:
Let's remove this in favor of Monaco's built in python mode